### PR TITLE
batches: Omit changelog for 3.37

### PR DESCRIPTION
**Pull request by eseliger**
## Test plan

Visual testing in storybook.

### CODE REVIEW

File: client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx [hunk 1]
```diff
@@ -2,24 +2,20 @@ import classNames from 'classnames'
...
             <CardBody>
-                <h4>Batch Changes updates in version 3.36</h4>
+                <h4>Batch Changes updates in version 3.37</h4>
                 <ul className="mb-0 pl-3">
-                    <li>
-                        <Link to="/help/admin/config/batch_changes#forks" rel="noopener" target="_blank">
-                            Batch Changes now supports pushing changesets to forked repositories.
-                        </Link>
-                    </li>
+                    <li>Nothing noteworthy this time.</li>
```
Comment (courier-new) [inline client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx:18]: Aw man, put this way, it sounds so depressing. 😂 Could we allude to upcoming changes instead, maybe? This is probably way too cheesy but:
```suggestion
                    <li>Nothing noteworthy this time, but rest assured, there's some big stuff cooking.</li>
```

File: client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx [hunk 2]
```diff
@@ -5,33 +5,22 @@ import { CardBody, Card, Link } from '@sourcegraph/wildcard'
...
-                        <BatchChangesUnlicensedAlert />
-                    </div>
-                    <div className="col-12 col-md-6 mb-3">
-                        <BatchChangesChangelogAlert />
-                    </div>
-                </>
-            )}
+            <div className="col-12 col-md-6 mb-3">
+                <BatchChangesUnlicensedAlert />
+            </div>
```
Comment (courier-new) [inline client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx:23]: Is it worth leaving this code commented out so it's even easier to restore for the next version?

**Review by courier-new — APPROVED** [review]

File: client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx (hunk 2 above)
Comment (eseliger) [reply client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx:23]: people tend to yell at me when they see commented out code so I opted for git reverting later 😆

File: client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx (hunk 1 above)
Comment (eseliger) [reply client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx:18]: It's not actually rendered in the app, so I guess it doesn't matter?

Comment (courier-new) [reply client/web/src/enterprise/batches/list/BatchChangesChangelogAlert.tsx:18]: Ohhh I see, we *only* use it in `BatchChangesListIntro`? Yeah don't worry about it then. 😂

File: client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx (hunk 2 above)
Comment (courier-new) [reply client/web/src/enterprise/batches/list/BatchChangesListIntro.tsx:23]: Oh no! Okay I promise I won't yell at you either way. 😂

**Pull request merged**